### ISSUE TITLE
groups_get_group( 'group_id=...)

### ISSUE THREAD

**Issue by johappel** [boonebgorges/buddypress-docs#556]
Hi,

the plugin needs an urgent fix

```
`groups_get_group( 'group_id=' . $id )`  does not work since buddypress 2.7
```

would you please fix it with  `groups_get_group( $id )`

**Comment by dcavins** [comment]
Thanks for the report! Yes, it's actually a bug we introduced in BP2.7. We'll issue a fix soon.

**Comment by dcavins** [comment]
This was addressed in 1.9.2. Thanks!
